src/objc_glue_stubs.c

```diff
@@ -17,6 +17,11 @@ class Type;
 class TypeFunction;
 struct elem;
 
+void objc_initSymbols()
+{
+    // noop
+}
+
```

Comment:
You remove the file from the Makefile below. Are you sure this modification is needed then?

Reply:
That's a different file, `obc_stubs.c`, this one is still in there.  Guess it was renamed at some point.